src/writeData/components/WriteDataHelperBuckets.tsx

```diff
@@ -35,10 +35,28 @@ const WriteDataHelperBuckets: FC<Props> = ({
   className = 'write-data--details-widget-title',
   useSimplifiedBucketForm = false,
 }) => {
+  const {bucket, changeBucket} = useContext(WriteDataDetailsContext)
+  const isSelected = (bucketID: string): boolean => {
```

Comment:
I think we need `useCallback` here

Reply:
why is that? I just moved this function up in the file - it wasn't using `useCallback` before